tgui/packages/tgui/interfaces/PreferencesMenu/QuirksPage.tsx

```diff
@@ -203,6 +204,14 @@ export const QuirksPage = (props, context) => {
             }
           }
 
+          if (chosenSpecies in speciesquirkBlacklist) {
+            if (
+              speciesquirkBlacklist[chosenSpecies].indexOf(quirk.name) !== -1
+            ) {
+              return `This is incompatible with the species ${data.species[chosenSpecies].name}!`;
```

Comment:
Change "with the species [name]" to just "[name but plural]". Plasmamen etc. I'm fairly sure we have a var for that, but if we don't then add one.

In fact, replace the whole thing with:

"[quirk name] is not compatible with [species name, plural]"

Reply:
Plural form is not communicated to tgui, you would need to add that to `/datum/preference/choiced/species/compile_constant_data()`